### ISSUE TITLE
Hetzner provider: failed to create workspace: container exited with status 14

### ISSUE THREAD

**Issue by johnnypea** [daytonaio/daytona-provider-hetzner#5]
When I tried to run this repository https://github.com/johnnypea/devcontainer-compose using Hetzner provider I got this error:

```bash
 devcontainer-compose | Image pulled successfully
 devcontainer-compose | Reading devcontainer configuration...
 devcontainer-compose | Running initialize command...
FATA[0229] failed to create workspace: container exited with status 14 
```

Daytona version v0.46.1

**Comment by Tpuljak** [comment]
@johnnypea transferred the issue here.

What's the version of the provider you're using? You can check with `daytona provider ls`.

**Comment by johnnypea** [comment]
@Tpuljak `hetzner-provider v0.2.2`

**Comment by Tpuljak** [comment]
@johnnypea the fix was released in [v0.47.0](https://github.com/daytonaio/daytona/releases/tag/v0.47.0). Please also update all providers with `daytona provider update --all`.
